added csv version of skyhookv2 schema.

KDahlgren opened the pull request.

ivotron committed:
Adds github action workflow for CI test

ivotron committed:
Makes it a 1-osd vstart

ivotron committed:
Adds README

[skip ci]

ivotron committed:
Downloads data directly and installs deps

ivotron committed:
Switches to ubuntu bionic

jlefevre reviewed (commented):
lgtm.  i will approve+merge after other pr requests finish building

jlefevre reviewed (commented):
great, thanks.

KDahlgren committed:
added csv version of skyhookv2 schema.

KDahlgren committed:
added csv version of skyhookv2 schema.

KDahlgren committed:
identifier changes plus whitespace clean up.

KDahlgren committed:
identifier changes plus whitespace clean up.

KDahlgren committed:
edited comment in fbs file. note that i actually
recompiled the schema for the generated file, just to be safe.
evidently nothing in the _generated.h file changed.

KDahlgren committed:
edited comment in fbs file. note that i actually
recompiled the schema for the generated file, just to be safe.
evidently nothing in the _generated.h file changed.

ivotron committed:
Brings back micro-osd

Builds only CLS and creates minimal ceph image to test

ivotron committed:
Fixes dependency specification in workflow

ivotron committed:
Adds note to README

[skip ci]

ivotron committed:
Updates path to cls folder

jlefevre committed:
Merge pull request #38 from ivotron/ci-popper

adds github action workflow for CI test

KDahlgren committed:
Merge branch 'skyhook-skyhookv2_csv' of github.com:uccross/skyhook-ceph into skyhook-skyhookv2_csv
